### ISSUE TITLE
gsettings schema install with flatpak

### ISSUE THREAD

**Issue by aral** [elementary/docs#130]
In the [State Saving](https://docs.elementary.io/develop/apis/state-saving) section, you’re instructed to use `ninja install` to ensure that the settings schema is installed and compiled.

Given you can’t do that with a flatpak install, would it make sense to include a link here to the equivalent flatpak configuration instructions?

(I’m assuming something like this? https://github.com/flathub/org.gnome.Mahjongg/pull/10/commits/7ae0aa0960317c86f9186af3147f5ff29415e485)

**Comment by cassidyjames** [comment]
I _believe_ this is already handled by the build system in the Flatpak packaging process; are you seeing otherwise? If you're developing "natively" e.g. not building with Flatpak every time, then you _do_ need to ensure the settings schema is installed natively as well, and `ninja` would be the way to do that. Perhaps it's a matter of cleaning up language around when to use `ninja` and when to use `flatpak-builder` for development.

**Comment by aral** [comment]
> Perhaps it's a matter of cleaning up language around when to use `ninja` and when to use `flatpak-builder` for development.

Yeah, I think this would help newcomers (it would have helped me a few weeks ago) :)
